style.css

```diff
@@ -1,624 +1,995 @@
+}
+
+.more-of-my-works {
+  padding-left: 16px;
+}
+
+.feature-work {
+  display: none;
+}
+
+.tools-used-here {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding-left: 16px;
+  gap: 10px;
+  margin-top: 0;
+  margin-bottom: 10px;
+  padding-inline-start: 0;
+  list-style: none;
+}
+
+.first-project-tools {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding-left: 16px;
+  gap: 10px;
+  margin-top: 0;
+  margin-bottom: 10px;
+  padding-inline-start: 0;
+  list-style: none;
+}
+
+.view-work {
+  background-color: #f55800;
+  margin-left: -16px;
+  margin-right: -16px;
+  width: calc(100% + 16px);
+  height: 40px;
+  text-align: center;
+  text-decoration-line: none;
+  padding: 10px;
+  color: #fff;
+}
+
+.recent-work-pic {
+  width: 100%;
+  object-fit: contain;
+}
+
+.recent-work-pic img {
+  width: 100%;
+  height: auto;
+  object-fit: fill;
+  display: block;
+}
+
+.recent-work-details {
+  width: 100%;
+  margin: 0;
+}
+
+#work-third {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#fourth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
+
+#fifth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/first-image.png");
+}
+
+#sixth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#seventh-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
+
+.work-title {
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  margin: 5px 0 5px 0;
+}
+
+.works-description {
+  margin: 5px 0 5px 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #091e42;
+}
+
+#personal-data {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 375px;
+  height: 1900px;
+  padding: 20px 20px;
+  background-color: #000;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+#my-cv-here {
+  background-image: url("./images/about_me_image.svg");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  height: 650px;
+  object-fit: contain;
+}
+
+.card-link {
+  text-decoration-line: none;
+  color: #fff;
+  background-color: #f55800;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  padding: 12px;
+  gap: 10px;
+  width: 171px;
+  height: 48px;
+}
+
+#my-cv-here a:active {
+  background: #e05304;
+}
+
+#my-cv-here a:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
```

Comment:
- [ ] The `skills section` style looks broken on the mobile version so please try your best to fix it 😃 👍 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/200010869-0bc184d0-8929-4e57-bef6-1f188120b267.png)
**Given design**
![image](https://user-images.githubusercontent.com/90222110/200011022-f86c5617-3b96-483c-af69-dac4414300e0.png)

Reply:
![skill-list](https://user-images.githubusercontent.com/32777408/200088897-66e5c2fa-4681-4fe8-9218-5807d11ab827.png)


This issue has been resolved.